Bottom: d223c5a3d9f0aa160866a46f9f440b4661272ac3
Top:    3215cffa1eaaf77685a00e3f7bfd446f8bafe66a
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-02-08 02:04:26 +0000

Refresh of improve-gl-stuff

---

diff --git a/src/hid/gtk/gtkhid-main.c b/src/hid/gtk/gtkhid-main.c
index abc7ddb..3332d7e 100644
--- a/src/hid/gtk/gtkhid-main.c
+++ b/src/hid/gtk/gtkhid-main.c
@@ -47,7 +47,7 @@
 
 #define PIXELS_PER_CIRCLINE 5.
 
-RCSID ("$Id: gtkhid-main.c,v 1.59 2008-12-27 19:06:04 djdelorie Exp $");
+RCSID ("$Id$");
 
 
 extern HID ghid_hid;
